### ISSUE TITLE
setting of early stopping is confused

### ISSUE THREAD

**Issue by hzaybr** [tianqitang1/ContigNet#2]
Hi, the paper said that the model was trained for 5000 epochs with early stopping criteria 0.00001 for 50 epochs, but I didn't find early stopping in your code, did it mean if loss didn't decrease to 0.00001 for consecutive 50 epochs, if not, could you describe how it works, and did the early stopping work on the test set? 
Thanks a lot!

**Comment by tianqitang1** [comment]
The code implemented early stopping was not synced to GitHub. The early stopping criteria means if the best loss does not improve by 0.00001 for 50 epochs, we stop the training. As for your second question, the early stopping is for training, I am not sure what did it work on the test set means?

**Comment by hzaybr** [comment]
The early stopping is for training, so the data only split into training set and testing set? Is there validation set? And could I ask that how many epochs did the early stopping take usually?
Thanks!

**Comment by tianqitang1** [comment]
Sorry for the late reply. A small portion of the training set (10%) was directly separated as validation for early stopping. The early stopping time varied but it usually stopped around 1-2k epochs for me.

**Comment by hzaybr** [comment]
Thank for the reply!
